plugins/catalog-backend/scripts/mock-db

```diff
@@ -0,0 +1,12 @@
+curl --location --request POST 'localhost:3003/locations' \
```

Comment:
Should probably have a shebang at the start of the file, e.g. `#!/usr/bin/env sh`.

And `chmod +x` the file :)

Could you add a newline between commands? And indent hanging lines, like

```
do \
  the \
  thing

and \
  then \
  this
```

that makes it easier to read

Reply:
👍